src/tests/Index.test.ts

```diff
@@ -24,12 +24,15 @@ const config = {
    // backgroundImage: 'https://i.pinimg.com/474x/d4/48/2b/d4482ba4e7ebdbff7b8ba73e7d39aceb.jpg',
  // backgroundImage:'https://www.fnordware.com/superpng/pnggrad16rgb.png',
  // backgroundImage:'https://png.pngtree.com/png-clipart/20200721/original/pngtree-design-scene-prototype-renderings-logo-sample-material-png-image_4913697.jpg',
- logoImage:'https://www.tutorialspoint.com/videotutorials/images/coding_ground_home.jpg',  
+ logoImage:'https://content.internetretailing.net/AcuCustom/Sitename/DAM/027/HSS-Hire-logo-3D2-1024x369.jpg',  
  backgroundColor: 'white',
     canvasType: CanvasType.SVG,
-    eyeFrameShape: EyeFrameShape.CIRCLE,
-    eyeBallShape: EyeBallShape.CIRCLE,
-    dataPattern: DataPattern.KITE,
+    eyeFrameShape: EyeFrameShape.SQUARE,
+    eyeBallShape: EyeBallShape.SQUARE,
+    // dataPattern: DataPattern.KITE,
+    rectangular: true,
+    widthSize: 5000,
```

Comment:
@hritikaggarwal31 please use more self-explanatory variable names instead of `widthSize` and `heightSize`